testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py

```diff
@@ -120,45 +124,59 @@ def test_verify_invalid_maskPassword_in_logs(params_from_base_test_setup, invali
     sg_client.create_user(sg_admin_url, sg_db, "autotest", password="password", channels=channels)
     sg_client.create_session(sg_admin_url, sg_db, "autotest")
 
-    replicator_authenticator = authenticator.authentication(username="autotest", password=invalid_password, authentication_type="basic")
+    replicator_authenticator = authenticator.authentication(username="autotest", password=invalid_password,
+                                                            authentication_type="basic")
     repl_config = replicator.configure(cbl_db, target_url=sg_blip_url, continuous=False,
                                        channels=channels, replicator_authenticator=replicator_authenticator)
 
     repl = replicator.create(repl_config)
     replicator.start(repl)
     replicator.wait_until_replicator_idle(repl, err_check=False)
-
-    verify_password_masked(liteserv_platform, log_file, invalid_password)
+    verify_password_masked(liteserv_platform, log_file, invalid_password, testserver, log_filename)
 
 
-def verify_password_masked(liteserv_platform, log_file, password):
-    if liteserv_platform.lower() == "android":
-        log_file = log_file.replace(" ", "\ ")
```

Comment:
I do not see this logic for android. Is this handle properly for log path?

Reply:
yes, there is no need to escape whitespace as we are encode in quote during grep command.

command = "grep '{}' {}/*.cbllog | wc -l".format(password, log_file)